addons/website/static/src/js/content/menu.js

```diff
@@ -233,4 +238,108 @@ publicWidget.registry.menuDirection = publicWidget.Widget.extend({
         $menu.addClass('dropdown-menu-' + alignment);
     },
 });
+
+publicWidget.registry.fadeOutHeader = publicWidget.Widget.extend({
+    selector: 'header.o_header_fade_out',
+
+    /**
+     * @override
+     */
+    start: function () {
+        this.scrollingDownwards = true;
+        this.hiddenHeader = false;
+        this.position = 0;
+        this.checkPoint = 0;
+        // Window Handlers
+        $(window).on('scroll', _.throttle(this._onWindowScroll.bind(this), 100));
+        setTimeout(this._onWindowScroll.bind(this), 0); // setTimeout to allow override with advanced stuff... see themes
```

Comment:
I can't find this advanced stuff and I don't see why a `setTimeout` would allow override ?

Reply:
I removed this lines